pudb/theme.py

```diff
@@ -469,57 +469,116 @@ def link(child: str, parent: str):
+            "highlighted var value": ("black", "dark green"),
+            "focused highlighted var label": ("black", "light green"),
+            "focused highlighted var value": ("black", "light green"),
+
+            "return label": ("white", "dark blue"),
+            "return value": ("black", "dark cyan"),
+            "focused return label": ("light gray", "dark blue"),
             # }}}
             # {{{ stack
-            "current frame name": (
-                add_setting("light gray", "bold"), "black"),
+            "stack": ("white", "black"),
+
+            "frame name": ("white", "black"),
+            "frame class": ("light blue", "black"),
+            "frame location": ("light cyan", "black"),
+
+            "current frame name": (add_setting("white", "bold"), "black"),
+            "current frame class": (add_setting("light blue", "bold"), "black"),
+            "current frame location": (add_setting("light cyan", "bold"), "black"),
+
+            "focused frame name": ("white", "dark blue"),
+            "focused frame class": ("white", "dark blue"),
+            "focused frame location": ("white", "dark blue"),
+
+            "focused current frame name": (
+                add_setting("white", "bold"), "dark blue"),
+            "focused current frame class": (
+                add_setting("white", "bold"), "dark blue"),
+            "focused current frame location": (
+                add_setting("white", "bold"), "dark blue"),
+            # }}}
+            # {{{ breakpoints view
+            "breakpoint": ("white", "black"),
+            "disabled breakpoint": ("dark gray", "black"),
+            "focused breakpoint": ("white", "dark blue"),
+            "focused disabled breakpoint": ("light gray", "dark blue"),
+            "current breakpoint": (add_setting("white", "bold"), "black"),
+            "disabled current breakpoint": (
+                add_setting("dark gray", "bold"), "black"),
+            "focused current breakpoint": (
+                add_setting("white", "bold"), "dark blue"),
+            "focused disabled current breakpoint": (
+                add_setting("light gray", "bold"), "dark blue"),
             # }}}
             # {{{ shell
-            "command line prompt": (
-                add_setting("white", "bold"), "black"),
-            "command line output": ("yellow", "black"),
-            "command line input": (
-                add_setting("white", "bold"), "black"),
+            "command line edit": ("white", "black"),
+            "command line prompt": (add_setting("white", "bold"), "black"),
+
+            "command line output": ("light green", "black"),
+            "command line input": ("white", "black"),
+            "command line error": (add_setting("light red", "bold"), "black"),
+
+            "command line clear button": (add_setting("white", "bold"), "black"),
+            "command line focused button": ("black", "light gray"),
```

Comment:
Perhaps make this button feel like other non-dialog selectable elements when focused?
```suggestion
            "command line focused button": ("white", "dark blue"),
```